apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/Slate.kt

```diff
@@ -13,7 +13,7 @@ data class Slate(
     @JsonValue val elements: List<Element> = emptyList()
 ) {
     fun flettInn(behandling: Behandling): Slate = when (behandling.revurderingsaarsak) {
-        RevurderingAarsak.ADOPSJON -> erstatt(this, BrevDataMapper.fra(behandling).second)
+        RevurderingAarsak.ADOPSJON -> erstatt(this, BrevDataMapper.brevData(behandling))
```

Comment:
Ser at omgjøring av farskap er litt med og litt ikke med i denne PR'en -- hadde vært lettere å følge med hvis det var alt eller ingenting for den revurderingsårsaken

Reply:
Godt poeng! Noterer det bak øyret til neste gong.